pom.xml

```diff
@@ -78,7 +78,15 @@
 					</processTypes>
 				</configuration>
 			</plugin>
-		</plugins>
+            <plugin>
```

Comment:
is this plugin used by implementation?

Reply:
No, it just updates when I clone the repository

Reply:
So, is it necessary or isn't?

Reply:
about this plugin, is still necessary?